src/components/MintSong/MintSong.tsx

```diff
@@ -100,7 +108,7 @@ const MintSong: React.FC = () => {
           // TODO: generate this, maybe not important for mvp
           "advAssetHash",
           // formData.adDuration returns number of days
-          formData.adDurationDays * 86400 // 1 Day == 86400 seconds
+          formData.adDurationDays ?? 3 * 86400 // 1 Day == 86400 seconds
```

Comment:
why 3 * 86400?

Reply:
Because it is set by default 3 and the ant form returns NaN instead of default 3 field

Reply:
oh something must be wrong with the intial value of the slider input i believe

Reply:
Yeah, can you check that, and update in this PR? Or maybe address in new issue

Reply:
i'll address it in another issue. Just open it up for me please

Reply:
Sure https://github.com/meta-web3-music/frontend/issues/93